src/app/footer/footer.component.html

```diff
@@ -0,0 +1,11 @@
+<footer id="stlFooter" fxLayout="row">
+  <nav role="navigation" fxLayout.gt-md="row" fxLayout.lt-md="column" fxLayoutGap.gt-md="1em">
+    <a fxHide.lt-md="true" class="stlFooterLink fontSizeNormal" routerLink="">Home</a>
```

Comment:
I don't think these need any value (i.e. no need for `="true"`) for the `fxHide` if it's `true`